Floofbot/Modules/Administration.cs

```diff
@@ -125,12 +127,12 @@ public class Administration : ModuleBase<SocketCommandContext>
             builder.Title = "⚖️ Warn Notification";
             builder.Description = $"You have recieved a warning in {Context.Guild.Name}";
             builder.AddField("Reason", reason);
-            builder.Color = Color.DarkOrange;
+            builder.Color = ADMIN_COLOR;
             await badUser.SendMessageAsync("", false, builder.Build());
 
             builder = new EmbedBuilder();
             builder.Title = (":shield: User Warned");
-            builder.Color = Color.DarkOrange;
+            builder.Color = ADMIN_COLOR;
```

Comment:
To address in a future PR: actually decide on the colours, since this is the exact same colour as the embeds for the Fun module.